browser/ui/webui/playlist_page_handler.cc

```diff
@@ -72,6 +76,20 @@ void PlaylistPageHandler::AddMediaFilesFromPageToPlaylist(const std::string& id,
                                                                   url.spec());
 }
 
+void PlaylistPageHandler::AddMediaFilesFromOpenTabsToPlaylist(
+    const std::string& playlist_id) {
+  auto* browser = chrome::FindBrowserWithWebContents(web_contents_);
+  DCHECK(browser);
+  auto* tab_strip_model = browser->tab_strip_model();
+  for (auto i = 0; i < tab_strip_model->count(); i++) {
+    if (auto* contents = tab_strip_model->GetWebContentsAt(i);
+        contents != web_contents_) {
+      GetPlaylistService(profile_)->RequestDownloadMediaFilesFromContents(
+          playlist_id, contents);
```

Comment:
this approach to enumerate only browser-bound web_contents is not pretty, but as a wip solution for the feature development it's ok.
I don't know what the final result we want so I'm not sure what to recommend here, but I hope this will be transformed into something more reliable in future.

Reply:
Yeah. I agree. There's no entry point for now and this is all experimental. We'll see :)